tests/bundle/SystemInfo/Collector/ServicesSystemInfoCollectorTest.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace Ibexa\Tests\Bundle\SystemInfo\SystemInfo\Collector;
+
+use Ibexa\Bundle\SystemInfo\SystemInfo\Collector\ServicesSystemInfoCollector;
+use Ibexa\Bundle\SystemInfo\SystemInfo\Value\ServicesSystemInfo;
+use Ibexa\SystemInfo\Service\Service;
+use Ibexa\SystemInfo\Service\ServiceProviderInterface;
+use PHPUnit\Framework\TestCase;
+
+class ServicesSystemInfoCollectorTest extends TestCase
+{
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject
+     */
+    private $serviceProviderMock;
+
+    /**
+     * @var \PHPUnit\Framework\MockObject\MockObject
+     */
+    private $serviceMock;
+
+    /**
+     * @var \Ibexa\Bundle\SystemInfo\SystemInfo\Collector\ServicesSystemInfoCollector
+     */
+    private $serviceCollector;
+
+    protected function setUp(): void
+    {
+        $this->serviceProviderMock = $this->createMock(ServiceProviderInterface::class);
+        $this->serviceMock = $this->createMock(Service::class);
+
+        $this->serviceCollector = new ServicesSystemInfoCollector($this->serviceProviderMock);
+    }
+
+    /**
+     * @covers \Ibexa\Bundle\SystemInfo\SystemInfo\Collector\ServicesSystemInfoCollector::collect()
```

Comment:
`@covers` annotation should not be used on method level:
> This annotation can be added to the docblock of the test class or the individual test methods. The recommended way is to add the annotation to the docblock of the test class, not to the docblock of the test methods.

Reason for this is that when code coverage is actually used, method-level declaration limits the scope to the method itself, preventing code coverage checks in any methods called by the method in question (i.e. if `collect` method calls any private method, it would appear as if it was not executed).